src/tpt_reports/tpt_reports.py

```diff
@@ -141,10 +141,10 @@ def main() -> None:
                 + "debug, info, warning, error, and critical.",
             ),
             # Issue #30 - Remove 3rd party reference from arguments
-            # TODO: Define a generic ID format to replace SERVICENOW_ID and provide validation.
+            # TODO: Define a generic ID format to replace ASSESSMENT_ID and provide validation.
```

Comment:
These two comment lines can be deleted since this PR resolves #30.

Reply:
Resolved in 7e32d31